_includes/use-cases/use-case-consumption-create-and-share-a-relationshipattribute.md

```diff
@@ -0,0 +1,25 @@
+{{properties.description}}
+
+{% include properties_list.html %}
+
+[RelationshipAttributes]({% link _docs_integrate/data-model-overview.md %}#relationshipattribute) are always associated with a [Relationship]({% link _docs_integrate/data-model-overview.md %}#relationship) between two Identities.
+Consequently, in contrast to [IdentityAttributes]({% link _docs_integrate/data-model-overview.md %}#identityattribute), there cannot be unshared RelationshipAttributes.
+Instead, you and your peer will always each have a [LocalAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute) with the same RelationshipAttribute as `content` and which only differs in the `shareInfo.peer` property.
+Thus, wanting to create a new RelationshipAttribute, a [Request]({% link _docs_integrate/data-model-overview.md %}#request) is sent to the peer via [Message]({% link _docs_integrate/data-model-overview.md %}#message).
+Only if the peer accepts this Request, the RelationshipAttribute will be created at their side.
+Once you receive the [Response]({% link _docs_integrate/data-model-overview.md %}#response), a LocalAttribute with the same `content` will be created at your side.
+
+## Parameters
+
+- The `content` of the RelationshipAttribute you want to create, following the description from the [data model]({% link _docs_integrate/data-model-overview.md %}#relationshipattribute), except for the `owner`, which is automatically set to your address
```

Comment:
We normally capitalise "Address" in this context (e.g. compare with https://enmeshed.eu/integrate/data-model-overview#localattributeshareinfo).

Same for other locations here and other use cases